src/Bookings.js

```diff
@@ -12,7 +13,7 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults fakeBookings={fakeBookings} />
```

Comment:
You know, your search component doesn't need to know if those bookings are fake or not. To what this component concern, it will receive some bookings and doesn't matter if they are fake or not. With that said, what about rename this props to just `bookings`?.